src/cli.ts

```diff
@@ -16,19 +28,32 @@ module.exports = function cli(commands) {
 
     .command(commands)
 
-    .option('verboseErrors', builders.verboseErrors)
-    .option('suppressWarnings', builders.suppressWarnings)
-
+    .option('verboseErrors', {
+      alias: ['v', 'verbose'],
+      type: 'boolean',
+      describe: 'Display original errors',
+      required: false,
+      default: false,
+    })
+    .option('suppressWarnings', {
+      alias: 'w',
+      type: 'boolean',
+      describe: 'Suppress warnings',
+      required: false,
+      default: false,
+    })
```

Comment:
There should be a newline between these two now that they span multiple lines, but also why are we not using builders here?

Reply:
I did it this way because options is typed by yargs and it does not accept the custom object we created. 
![Screen Shot 2021-02-01 at 2 28 22 PM](https://user-images.githubusercontent.com/58123480/106526023-c0104e00-6499-11eb-8673-90e8f1b577b6.png)

Reply:
![Screen Shot 2021-02-01 at 5 40 06 PM](https://user-images.githubusercontent.com/58123480/106540455-d5df3c80-64b4-11eb-815c-c649ae303af5.png)

I fixed the above error but it is still best to keep this implementation because changing it would result in unknown typing for the middleware and result in the picture above. Currently it is very nicely inferred. 

Commit where I found out how to type yargs: 868dab5